src/index.html

```diff
@@ -1,13 +1,66 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="sans-serif"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page">
+
+    <div class="page__card" data-qa="card">
+
+      <header class="page__picture">
+        <img
+          class="page__picture-image"
+          src="images/imac.jpeg"
+          alt="mac"
+        >
+      </header>
+
+      <h1 class="page__item-name item-name">
+        Моноблок APPLE A1419 iMac
+        27" Retina 5K (MNED2UA/A)
+      </h1>
+
+      <p class="page__code code">
+        Код товара: 195434
+      </p>
+
+      <div class="page__item-rate">
+        <ul class="page__stars-row page__stars-row--4">
+          <li class="page__star"></li>
+          <li class="page__star"></li>
+          <li class="page__star"></li>
+          <li class="page__star"></li>
+          <li class="page__star"></li>
+        </ul>
```

Comment:
have you reused this from the previous task? why not divs?

Reply:
i used 'ul' because the task requirement is to use semantic, but i can change to div, no probs